Cargo.toml

```diff
@@ -25,6 +25,11 @@ homepage = "https://www.availproject.org/"
 avail-core = { git = "https://github.com/availproject/avail-core", tag = "node-v2230-rc2", default-features = false, features = [ "runtime"] }
 kate = { git = "https://github.com/availproject/avail-core", tag = "node-v2230-rc2", default-features = false }
 kate-recovery = { git = "https://github.com/availproject/avail-core", tag = "node-v2230-rc2", default-features = false  }
+ssz-rs = { package = "ssz_rs", version = "0.9.0" }
+alloy-primitives = { version= "0.7.7", features=[ "serde"] }
+common = {git = "https://github.com/a16z/helios.git"}
```

Comment:
nit: I'd alias these packages as `helios-common` and `helios-consensus-core`. Otherwise, it will get confusing.

You can do this by:

```toml
helios-common = {git = "https://github.com/a16z/helios.git", package = "common"}
```